### ISSUE TITLE
Windows Defender flags a specific file as Trojan Virus whenever CCAe is run.

### ISSUE THREAD

**Issue by djjohnsongeek** [ThePacielloGroup/CCAe#164]
**Describe the bug**
Windows Defender flags a specific file as Trojan Virus whenever CCAe is run and an "color dropper" button is clicked.
Path to file: C:Users\username\AppData\Local\Temp\0ac98240-6c8f-42ba-828a-679f55030e87.tmp.exe
It quarantines the file right away.

**Desktop/Application Info**
 - OS: Windows 10 64bit
 - OS: Version 10.0.18362 Build 18362
 - CCAE Verision: 3.01

**To Reproduce**
Start CCAe, click any of the "color dropper" icons.

**Expected behavior**
Magnified color select cursor to appear.

**Screenshots**
![image](https://user-images.githubusercontent.com/43294314/78562599-a9e92100-77e7-11ea-9e6e-3100037cda7d.png)

**Additional context**
Just installed windows updates,

**Comment by ferllings** [comment]
I think that's related to the new picker process, which now launch/use an embedded (native) app

**Comment by djjohnsongeek** [comment]
Hmm, It was working last week with no issues or Window's Defender warnings.
Must be a windows Defender update that is now causing it. Is there a way that I can contact the developers of embedded app?

**Comment by ferllings** [comment]
I built the app myself, but it's based on this code:

https://github.com/mockingbot/mb_colorpicker_desktop_native

**Comment by djjohnsongeek** [comment]
I would like to [submit the specific file to Microsoft](https://www.microsoft.com/en-us/wdsi/filesubmission) to see if they can give some indication why is is being flagged. Is this okay with you?

**Comment by ferllings** [comment]
Sure

**Comment by djjohnsongeek** [comment]
Thanks! I will reopen this issue with more information if Microsoft gets back to me.

**Comment by patrickhlauke** [comment]
randomly, can confirm that CCA now fails (i hadn't even noticed the notifications, as i have them set to "do not disturb", but did notice the picker was failing - happens when pressing the picker button)

![image](https://user-images.githubusercontent.com/895831/78574084-6ad7c180-7821-11ea-87f9-3b381be8a4e4.png)

**Comment by djjohnsongeek** [comment]
Yeah, that is the exact same behavior that I am experiencing. I submitted the file, it is under review by Microsoft. I've re-opened the issues, since its not just me.

**Comment by n-chardon** [comment]
I have the same issue : the [64 bit portable version](https://github.com/ThePacielloGroup/CCAe/releases/download/v3.0.1/CCA-Portable-64.exe) is flagged as malware by Windows Defender: 
![virus-alert](https://user-images.githubusercontent.com/14290657/78576707-57caee80-782d-11ea-988e-191b4b7cbb34.png)

**Comment by djjohnsongeek** [comment]
Microsoft responded. After following their instructions below everything is working as expected for me.
---
![image](https://user-images.githubusercontent.com/43294314/78582572-7a481200-7803-11ea-937a-173a3eff2fd0.png)

**Comment by djjohnsongeek** [comment]
No problem, happy to help out!
@patrickhlauke
